Services/Certificate/README.md

```diff
@@ -395,6 +411,7 @@ listening to:
 |------------------|----------------------|-------------------------------------------------------|
 | updateStatus     | Services/Tracking    | This event will be thrown by the Learning Progress    |
 | migrateUserCertificate | Services/Certificate | These are custom events to add a new user certificate |
+| deleteUser | Services/User | This event is emitted when the user is deleted. Every relevant data in the certificates will be deleted for this user. Due the privacy regulations of ILIAS this data is **not** recoverable |
```

Comment:
Does this really belong to this PR?

Reply:
You are absolutely right :+1: I will remove this.